src/SourceBuild/content/repo-projects/fsharp.proj

```diff
@@ -19,9 +19,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <RepositoryReference Include="arcade" />
-    <RepositoryReference Include="msbuild" /> 
-    <RepositoryReference Include="source-build-externals" Condition="'$(DotNetBuildSourceOnly)' == 'true'" />
+    <RepositoryReference Include="msbuild" />
```

Comment:
This should be source-only

Reply:
Fixed in https://github.com/dotnet/installer/pull/18557/commits/4d7ac80f5ef401c4d2cab7e7bcd339811db760c6